src/exercise/client/src/Components/BottomButtons/BottomButtons.js

```diff
@@ -0,0 +1,84 @@
+import "./BottomButtons.css";
+import { Button } from "monday-ui-react-core";
+import { useState } from "react";
+
+// sort tasks by date they created
+function sortTime(ascTime) {
+  const taskSections = document.body.querySelectorAll("div.taskSection");
+  let tmpTaskSections = [...taskSections];
+  tmpTaskSections.sort((a, b) => {
+    a = a.querySelector("div.task").querySelector("p.time").id;
+    b = b.querySelector("div.task").querySelector("p.time").id;
+    if (ascTime) {
+      return new Date(a) - new Date(b);
+    }
+    return new Date(b) - new Date(a);
+  });
+  for (let i = 0; i < tmpTaskSections.length; i++) {
+    taskSections[i].parentNode.appendChild(tmpTaskSections[i]);
+  }
+}
+
+// sort tasks by name
+function sortAZ(ascAZ) {
+  const taskSections = document.body.querySelectorAll("div.taskSection");
+  let tmpTaskSections = [...taskSections];
+  tmpTaskSections.sort((a, b) => {
+    a = a.querySelector("div.task").querySelector("p").innerText;
+    b = b.querySelector("div.task").querySelector("p").innerText;
+    if (ascAZ) {
+      return a.localeCompare(b);
+    }
+    return b.localeCompare(a);
+  });
+  for (let i = 0; i < tmpTaskSections.length; i++) {
+    taskSections[i].parentNode.appendChild(tmpTaskSections[i]);
+  }
+}
+
+export default function BottomButtons({ deleteAll }) {
+  const [ascTime, setAscTime] = useState(false);
+  const [ascAZ, setAscAZ] = useState(true);
+
+  // clear all tasks
+  async function clearAll() {
+    const taskSections = document.body.querySelectorAll("div.taskSection");
```

Comment:
another thing, if for some reason you do want to access some dom elements use useRef! you don't need those query selectors anymore in most cases
they will just cause you troubles